Samples/NetworkMediaStreamer/RTPMediaStreamer/build/RTPMediaStreamer.vcxproj

```diff
@@ -37,33 +37,33 @@
     <ProjectGuid>{763269FF-37EB-4A0A-B152-0924CFA81FFF}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>VideoStreamer</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.22000.0</WindowsTargetPlatformVersion>
     <ProjectName>RTPMediaStreamer</ProjectName>

```

Comment:
Suggest: Add ARM64 build configurations, to be uniform with our other samples

Reply:
will take this up as a separate PR in interest of unblocking ROS work.